src/muya/lib/index.js

```diff
@@ -321,6 +324,14 @@ class Muya {
         this.container.classList.add('ag-show-quick-insert-hint')
       }
     }
+
+    const editorLineWidth = options['editorLineWidth']
```

Comment:
I think it's not a good idea to control the editor width in muya. Muya should be more general. The control width should be controlled by external users, that is, in the application using muya to control the edit width.

Reply:
I moved it into the renderer but the value is not automatically reloaded when changed in settings because vue don't listen to computed properties. I think you know a solution for this and can fix this afterwards.

Reply:
I think you can move the code to the renderer first, and then add the existed bug description to the PR description. I can pull it to the local to see what the problem is.